.changeset/lemon-cougars-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@redocly/cli": patch
```

Comment:
Should it include `@redocly/openapi-core` bump too?

Reply:
No as all the changes are scoped to the `cli` package.